src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,70 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 9;
+    private K key;
+    private V value;
+    private int size = 0;
+    private StorageImpl[] storArr = new StorageImpl[MAX_SIZE];
+
+    public StorageImpl() {
+    }
```

Comment:
All non-constant fields should be initialized in the constructor (from common mistakes)
```suggestion
    private int size;
    private StorageImpl[] storArr;

    public StorageImpl() {
    size = 0;
    storArr = = new StorageImpl[MAX_SIZE];
    }
```